test/repositories/DataCacheRepositorySpec.scala

```diff
@@ -207,16 +205,14 @@ class DataCacheRepositorySpec extends AnyWordSpec with MockitoSugar with Matcher
         ).toFuture()
 
         response <- dataCacheRepository.remove(pstr)
-        firstRetrieved <- dataCacheRepository.get(pstr)
```

Comment:
@PavelVjalicin  here when we try to get a pstr which is removed in first call,, locally it works fine by not fetching.. but in jenkins build it fetches. only this test is failing..